be/src/exec/tablet_sink_colocate_sender.h

```diff
@@ -0,0 +1,89 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <memory>
+#include <queue>
+#include <set>
+#include <string>
+#include <thread>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+#include "common/object_pool.h"
+#include "common/status.h"
+#include "common/tracer.h"
+#include "exec/data_sink.h"
+#include "exec/tablet_info.h"
+#include "exec/tablet_sink_index_channel.h"
+#include "exec/tablet_sink_sender.h"
+#include "gen_cpp/Types_types.h"
+#include "gen_cpp/doris_internal_service.pb.h"
+#include "gen_cpp/internal_service.pb.h"
+#include "runtime/mem_tracker.h"
+#include "util/compression/block_compression.h"
+#include "util/raw_container.h"
+#include "util/ref_count_closure.h"
+#include "util/reusable_closure.h"
+#include "util/threadpool.h"
+
+namespace starrocks {
+
+class MemTracker;
+class RuntimeProfile;
+class RowDescriptor;
+class TupleDescriptor;
+class ExprContext;
+class TExpr;
+
+namespace stream_load {
```

Comment:
how about simplify the namespace, it's not only a `stream_load` anymore ?

Reply:
Can I fix this in the later pr ? I will make the colocate sender work in the next pr.